drivers/SmartThings/zigbee-switch/src/test/test_all_capability_zigbee_bulb.lua

```diff
@@ -189,6 +189,32 @@ test.register_message_test(
     }
 )
 
+test.register_coroutine_test(
+  "Max and min mired reports should generate a range report after both are received",
+  function ()
+    test.socket.zigbee:__queue_receive({
+      mock_device.id,
+      ColorControl.attributes.ColorTempPhysicalMinMireds:build_test_attr_report(mock_device, 555) -- ~1800 K
```

Comment:
I believe these are flipped, so a good test value for `ColorTempPhysicalMinMireds` is 154 which is roughly 6500K and a good value for `ColorTempPhysicalMaxMireds` is 555 which is roughly 1800K

Reply:
i.e. the min mireds corresponds to the max kelvin and vice versa

you are absolutely right. what cursed units